sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py

```diff
@@ -193,9 +223,13 @@ async def run(args):
         else:
             clients = [create_client(args)]
             tasks = [asyncio.ensure_future(
-                clients[0].receive(
-                    on_event_received,
+                clients[0].receive_batch(
+                    on_event_batch_received,
                     prefetch=args.link_credit,
+                    max_batch_size=args.max_batch_size
+                ) if args.max_batch_size else clients[0].receive_batch(
+                    on_event_received,
+                    **kwargs_dict
```

Comment:
same comment as above:

1. `else clients[0].receive()`
2. we could reuse dict_args variable for receive_batch as well